htp/htp_config_private.h

```diff
@@ -345,6 +345,9 @@ struct htp_cfg_t {
 
     /** max memory use by a the lzma decompressor. */
     size_t lzma_memlimit;
+
+    /** max output size for a compression bomb. */
+    int64_t compression_bomb_limit;
```

Comment:
I think this size is too big. We won't ever set it to the max. I know size_t also is 64bit (on 64bit archs) and I am generally not happy with how much it is used in the htp API. We should probably make this a uint32_t.

Reply:
OK, I use `int32_t` so as not to have signedness warning and I check for overflow in `htp_config_set_compression_bomb_limit`